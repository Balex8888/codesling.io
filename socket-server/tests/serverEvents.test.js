import {
  serverInitialState,
  serverChanged,
  serverLeave,
} from '../src/serverEvents';

// Expects all serverEvents to be functions
test('serverInitialState should be a function', () => {
  expect(typeof serverInitialState).toMatchSnapshot();
});

test('serverChanged should be a function', () => {
  expect(typeof serverChanged).toMatchSnapshot();
});

test('serverLeave should be a function', () => {
  expect(typeof serverLeave).toMatchSnapshot();
});
