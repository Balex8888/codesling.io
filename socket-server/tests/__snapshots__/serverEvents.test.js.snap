// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serverChanged should be a function 1`] = `"function"`;

exports[`serverInitialState should be a function 1`] = `"function"`;

exports[`serverLeave should be a function 1`] = `"function"`;
